philistine/mne/io.py

```diff
@@ -22,13 +21,14 @@
 # binary_int_16, ascii as future formats?
 
 
-supported_formats = {'binary_float32' : 'IEEE_FLOAT_32',
-                     'binary_int16'   : 'INT_16'}
+supported_formats = {'binary_float32': 'IEEE_FLOAT_32',
```

Comment:
Although PEP tools complain about it, the extra spaces here were intentional because *I* find the aligned key-value columns easier to read and it's my code.